modules/graceful/server_http.go

```diff
@@ -17,7 +20,7 @@ func newHTTPServer(network, address, name string, handler http.Handler) (*Server
 		ReadTimeout:    DefaultReadTimeOut,
 		WriteTimeout:   DefaultWriteTimeOut,
 		MaxHeaderBytes: DefaultMaxHeaderBytes,
-		Handler:        handler,
+		Handler:        h2c.NewHandler(handler, &http2.Server{}),
```

Comment:
What is the purpose of including h2c? To me, this deserves a comment.

Reply:
Reverted it. I think it is a legacy of experiments and is unnecessary now, related to https://gitea.com/gitea/act_runner/pulls/4.